src/Sarif.Viewer.VisualStudio/overview.md

```diff
@@ -0,0 +1 @@
+The Microsoft SARIF Viewer extension for Visual Studio provides a convenient UI for analyzing static analysis log files and taking action on the items they contain. The Viewer intgrates with the Visual Studio environment, displaying a list of analysis results in the Error List and result details in a dockable tool window. Embedded target files can be opened right in the text editor with the error line highlighted. If the target file isn't embeeded in the log file, the Viewer will prompt you to open a local copy.
```

Comment:
>Viewer [](start = 169, length = 6)

Should be lower case

Reply:
I'm capitalizing it as it's part of the full name, "SARIF Viewer."

Reply:
Added "SARIF"